src/components/Loans/OpenLoans/components/OpenLoansSubHeader/OpenLoansSubHeader.js

```diff
@@ -135,7 +137,8 @@ class OpenLoansSubHeader extends React.Component {
     // and values being the associated loan properties -- e.g. { uuid: {loan}, uuid2: {loan2} }. This makes
     // it a little complicated to determine whether any loan in checkedLoans has a particular property -- like
     // an item that's been declared lost
-    const onlyLostItemsSelected = !Object.values(checkedLoans).find(loan => loan?.item?.status?.name !== 'Declared lost');
+    const onlyLostItemsSelected = hasEveryLoanItemStatus(checkedLoans, itemStatuses.DECLARED_LOST);
+    const onlyClaimedReturnedItemsSelected = hasEveryLoanItemStatus(checkedLoans, itemStatuses.CLAIMED_RETURNED);
```

Comment:
Ha -- I did almost exactly the same refactoring in the issue I'm working on. You got to the PR first, though! I'll just have to deal with the merge conflicts.